Packages/com.llealloo.audiolink/Editor/Scripts/AudioLinkManager.cs

```diff
@@ -96,4 +100,66 @@ public static void InstallUdonSharp()
         }
         #endif
     }
+
+    [InitializeOnLoad]
+    public class DefineManager
+    {
+        static DefineManager()
+        {
+            AddDefineIfMissing(EditorUserBuildSettings.selectedBuildTargetGroup, "AUDIOLINK");
+            Shader.EnableKeyword("AUDIOLINK_IMPORTED");
+        }
+
+        private static void AddDefineIfMissing(BuildTargetGroup buildTarget, string newDefine)
+        {
+            bool definesChanged = false;
+            string existingDefines = PlayerSettings.GetScriptingDefineSymbolsForGroup(buildTarget);
+            HashSet<string> defineSet = new HashSet<string>();
+
+            if (existingDefines.Length > 0)
+                defineSet = new HashSet<string>(existingDefines.Split(';'));
+
+            if (defineSet.Add(newDefine))
+                definesChanged = true;
+
+            if (definesChanged)
+            {
+                string finalDefineString = string.Join(";", defineSet.ToArray());
+                PlayerSettings.SetScriptingDefineSymbolsForGroup(buildTarget, finalDefineString);
+                Debug.LogFormat("Set Scripting Define Symbols for selected build target ({0}) to: {1}",
+                    buildTarget.ToString(), finalDefineString);
+            }
+        }
+    }
+    public static class SceneManager
```

Comment:
I was actually more talking about at the class level. I'd take this a step further and combine SceneManager and AudioLinkImportFix into one class, naming it appropriately. They both do scene-and-package (asset, basically) management, if you look at the existing code in this file.

Reply:
The define manager is reasonably separate in purpose tho, so that split is fine, I think.

Reply:
what name do you propose?